src/Symfony/Component/Process/PhpExecutableFinder.php

```diff
@@ -81,6 +81,10 @@ public function find(bool $includeArgs = true): string|false
             $dirs[] = 'C:\xampp\php\\';
         }
 
+        if ($herdPath = getenv('HERD_HOME')) {
+            $dirs[] = $herdPath . \DIRECTORY_SEPARATOR . 'bin';
```

Comment:
```suggestion
            $dirs[] = $herdPath.\DIRECTORY_SEPARATOR.'bin';
```

Reply:
@wouterj we should add that rule to our PHP-CS-Fixer config IMHO